src/main/java/com/monumental/services/GoogleMapsService.java

```diff
@@ -41,7 +46,7 @@ public String getAddressFromCoordinates(Double lat, Double lon) {
      * @param address - String for the address to geocode
      * @return Geometry - Google Maps Geometry containing the location data for the address
```

Comment:
Update the docstring here. We should probably add a docstring for `getAddressFromCoordinates` too.